testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/auditlog/AuditLogBootingSyslogTest.java

```diff
@@ -115,8 +115,7 @@ public void testSyslog() throws Exception {
         final BlockingQueue<SyslogServerEventIF> queue = BlockedSyslogServerEventHandler.getQueue();
         queue.clear();
         container.start();
-        long endTime = System.currentTimeMillis() + TimeoutUtil.adjust(5000);
-        waitForExpectedQueueSize(18, queue);
+        waitForExpectedQueueSize(1, queue);
```

Comment:
This is due to the fact that there was a bug in boot : we didn't clear the queue when writing log entries on status change. The 18 was due to leftovers from the previous operations (we didn't understand that at that  time). Now this is coherent with the AuditLogBootingLogTestCase.